detection-rules/link_credential_phishing.yml

```diff
@@ -6,11 +6,9 @@ severity: "high"
 source: |
   type.inbound 
   and any(body.links, 
-      all([beta.linkanalysis(.)],
-          .credphish.disposition == "phishing"
-          and .credphish.brand.confidence in ("medium", "high")
-      )
-  ) 
+      beta.linkanalysis(.).credphish.disposition == "phishing"
+          and beta.linkanalysis(.).credphish.brand.confidence in ("medium", "high")
```

Comment:
```suggestion
      and beta.linkanalysis(.).credphish.brand.confidence in ("medium", "high")
```

Reply:
Thanks!! 👍  I didn't see a missing paren, both files didn't surface any syntax highlights in the editor, but it's entirely possible :)